README.md

```diff
@@ -1,38 +1,92 @@
-![Path18 logo](logo.png)
+![Path18 logo](./public/images/logo.png)
 
-##### web app for informing young people of a variety of careers and paths :mortar_board:
+[![Build Status](https://travis-ci.org/facn2/Path18.svg?branch=master)](https://travis-ci.org/facn2/Path18)
+[![codecov](https://codecov.io/gh/facn2/Path18/branch/master/graph/badge.svg)](https://codecov.io/gh/facn2/Path18)
+
+**[Web app](https://path18.herokuapp.com/) for informing young people of a variety of careers and paths** :mortar_board:
+
+### Local Install
+
+Clone the repository.
+```
+git clone https://github.com/facn2/Path18.git
+```
+Install npm modules.
+```
+npm install
+```
+View on localhost.
+```
+npm start
+```
+Run tests.
+```
+npm test
+```
 
 ### Problem Statement:
-* Unexposure to variety of jobs and career paths for young people
-* Uncertainty around career prospects
+* Young people have little exposure to the variety of jobs and careers paths they could pursue.
+* Uncertainty around career prospects.
+* Little knowledge of the path to take to get a specific job.
 
 ### User Journey:
 As a user I can browse a variety of career paths and view detailed information about the ones that I am interested in.
 
 ### User Story:
 ##### As a user I can...
-* read welcome page and instructions on how to use the app
-* view card with job and decide if I like / dislike it
-* click on card to view more details
-* View list of jobs I like
-* click on job to view details on the job and career path
-* read details instructions for the app in the help page
+* read welcome page and instructions on how to use the app.
```

Comment:
capitalize first letters for consistency

Reply:
Yeah I agree :+1: